### PR TITLE
announcement pagination

### PR DESCRIPTION
**Pull request by wangqiang1208**
depends on #62

### CODE REVIEW

**Comment by wangqiang1208** [comment]
Hmm, think next step I will do course settings, then we can turn/off the pagination according to settings.

File: spec/features/course/announcement_pagination_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    end
+
+    it { is_expected.to have_selector('nav.pagination') }
+
+    it 'lists each announcement' do
+      course.announcements.page(1).each do |announcement|
+        expect(page).to have_selector('div', text: announcement.title)
+        expect(page).to have_selector('div', text: announcement.content)
+      end
+    end
```
Comment (lowjoel) [inline spec/features/course/announcement_pagination_spec.rb]: I think you need another test for page 2 as well.

Comment (lowjoel) [inline spec/features/course/announcement_pagination_spec.rb]: @wangqiang1208 please add the test, then this can be merged.

**Comment by wangqiang1208** [comment]
@lowjoel I've updated

**Comment by lowjoel** [comment]
:+1:

**Pull request merged**